aexpect/ops_windows.py

```diff
@@ -0,0 +1,578 @@
+# This Python file uses the following encoding: utf-8
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Intra2net AG and aexpect contributors
+# Authors : Samir Aguiar <samir.aguiar@intra2net.com>
+
+"""
+A list of frequent operations performed through remote sessions to linux platforms.
```

Comment:
Shouldn't this be `windows` platforms?

Reply:
Nice catch and a quick copy paste on my side, thanks!